sdks/python/apache_beam/testing/util.py

```diff
@@ -75,6 +75,25 @@ def __repr__(self):
   return InAnyOrder(iterable)
 
 
+def equal_to_per_window(expected_dict_window_to_its_elements):
```

Comment:
<!--new_thread; commit:1d8257d6902c54d33062c23cce99e06fbf42c41d; resolved:0-->
Please add a comment about explaining how it is used, and what are the arguments, similar to `contains_in_any_order`. Otherwise it would not be obvious to the users.

Reply:
Done.